ncm-dirperm/src/main/perl/dirperm.pm

```diff
@@ -31,6 +36,22 @@ sub Configure($$@) {
     # Define paths for convenience.
     my $base = "/software/components/dirperm";
 
+    my $filesystems = "/system/filesystems";
+    if ($config->elementExists($filesystems)) {
+        my $fs_tree = $config->getElement($filesystems)->getTree();
+        push(@configured_mounts, map {$_->{mountpoint}}
```

Comment:
the code belows relies on the fact that each path ends with a `/`. can you check and/or force this?